### PR TITLE
Confirmation resume updates

### PR DESCRIPTION
**Pull request by alvin-law**

### CODE REVIEW

File: app/client/views/confirmation/confirmation.html [hunk 1]
```diff
@@ -94,19 +94,21 @@
...
+              name="wantsLaptop"
+              ng-model="user.confirmation.wantsLaptop">
+              <option value="">Laptop Status</option>
+              <option value="No">I do not want a laptop</option>
+              <option value="Yes">I want a laptop</option>
+            </select>
+          </div>
+
+          <div class="field">
+            <label> Are you a US citizen? </label>
```
Comment (alvin-law) [inline app/client/views/confirmation/confirmation.html]: This isn't merged right, check the changes for the laptop prompt at a73ad34752553a34ece5b58396c051002c29d279

File: app/client/views/confirmation/confirmation.html [hunk 2]
```diff
@@ -322,15 +320,106 @@
...
+                href="https://go.hackmit.org/travel-receipts"
+                class="ui pink button">
+                Upload Receipt
+              </a>
+            </div>
+
+          </div>
+
+          <br>
+          <br>-->
```
Comment (alvin-law) [inline app/client/views/confirmation/confirmation.html]: Why did this whole chunk get added back in?

File: app/client/views/confirmation/confirmation.html [hunk 3]
```diff
@@ -322,15 +320,106 @@
...
 
           <div class="divided title">Legal</div>
 
           <div class="field">
             <p>
               <a
                 target="_blank"
-                href="http://hack.osu.edu/2018/pdf/2018_code_of_conduct.pdf">
+                href="https://docs.google.com/document/d/1axtqVIWyeQafHZpIFBM4Ks1VQSImYAos4PrQK4CfggI/edit?usp=sharing">
```
Comment (alvin-law) [inline app/client/views/confirmation/confirmation.html]: Linkable @ https://hack.osu.edu/2019/pdf/2019_code_of_conduct.pdf

File: app/client/views/confirmation/confirmationCtrl.js [hunk 4]
```diff
@@ -81,6 +81,15 @@ angular.module('reg')
...
             },
+            wantsLaptop: {
+              identifier: 'wantsLaptop',
+              rules: [
+                {
+                  type: 'empty',
+                  prompt: 'Please give us a laptop preference!'
+                }
+              ]
+            },
```
Comment (alvin-law) [inline app/client/views/confirmation/confirmationCtrl.js]: See changes done in a73ad34752553a34ece5b58396c051002c29d279

File: app/server/models/User.js [hunk 5]
```diff
@@ -87,16 +87,20 @@ var confirmation = {
   wantsHardware: Boolean,
   hardware: String,
 
-  wantsLaptop: Boolean,
+  wantsLaptop: {
+    type: String,
+    enum: {
+      values: 'Yes No'.split(' '),
+    }
+  },
```
Comment (alvin-law) [inline app/server/models/User.js]: Changes in a73ad34752553a34ece5b58396c051002c29d279

File: app/server/models/User.js [hunk 6]
```diff
@@ -113,16 +117,14 @@ var confirmation = {
...
-  /*
   receipt: String,
 
   hostNeededFri: Boolean,
   hostNeededSat: Boolean,
   genderNeutral: Boolean,
   catFriendly: Boolean,
   smokingFriendly: Boolean,
-  hostNotes: String,*/
+  hostNotes: String,
```
Comment (alvin-law) [inline app/server/models/User.js]: Why did we uncomment the parts related to hosting?

File: app/server/models/User.js [hunk 7]
```diff
@@ -87,16 +87,20 @@ var confirmation = {
...
-  resume: String,
-
-  usCitizen: {
-    type: Boolean,
-    required: true,
-    default: false,
+  resume: {
+  type: Boolean,
+  required: true,
+  default: false,
```
Comment (alvin-law) [inline app/server/models/User.js]: Indentation here

File: app/client/views/confirmation/confirmationCtrl.js [hunk 8]
```diff
@@ -214,7 +262,8 @@ angular.module('reg')
       }
 
       $scope.submitForm = function(){
-        if ($('.ui.form').form('validate form')){
+        uploadResume();
+        if ($('.ui.form').form('is valid')){
```
Comment (alvin-law) [inline app/client/views/confirmation/confirmationCtrl.js]: I think this undid Parshva's validation fixes?

File: app/client/views/confirmation/confirmationCtrl.js [hunk 9]
```diff
@@ -188,9 +196,49 @@ angular.module('reg')
...
+               rules: [
+                 {
+                   type: 'empty',
+                   prompt: 'Please choose a file for your resume.'
+                 }
+               ]
+           });
+       }
+       console.log(user.confirmation.resume);
+       console.log('submit');
```
Comment (alvin-law) [inline app/client/views/confirmation/confirmationCtrl.js]: Can we reduce some of the console logging?

File: app/client/views/confirmation/confirmationCtrl.js (hunk 4 above)
Comment (alvin-law) [reply app/client/views/confirmation/confirmationCtrl.js]: This chunk just gets removed but check what was done in that commit and make sure it's all good

File: app/client/views/confirmation/confirmation.html (hunk 2 above)
Comment (neilmckibben) [reply app/client/views/confirmation/confirmation.html]: We want all of the comments removed right

File: app/client/views/confirmation/confirmation.html (hunk 3 above)
Comment (neilmckibben) [reply app/client/views/confirmation/confirmation.html]: thanks

File: app/client/views/confirmation/confirmation.html (hunk 2 above)
Comment (alvin-law) [reply app/client/views/confirmation/confirmation.html]: Anything we already have, remove excess comments. Leave useful functionality commented

File: app/server/models/User.js (hunk 6 above)
Comment (neilmckibben) [reply app/server/models/User.js]: Not sure I will re-comment it.

File: app/client/views/confirmation/confirmationCtrl.js (hunk 9 above)
Comment (neilmckibben) [reply app/client/views/confirmation/confirmationCtrl.js]: Will remove all console logging once I'm confident it works

**Pull request merged**